instrumentation-core/spring/src/main/java/io/opentelemetry/contrib/spring/Application.java

```diff
@@ -0,0 +1,13 @@
+package io.opentelemetry.contrib.spring;
```

Comment:
@trask Is this the right package? The other instrumentation in `instrumentation-core` doesn't use the word contrib. If this is something separate from "official instrumentation", then we may need a contrib folder as well to group them together. But I guess this could just be `instrumentation`?

Reply:
👍 we aren't using the `contrib` terminology for things in this repo